### DISCUSSION TITLE
Help, command no worky

### DISCUSSION THREAD

**Discussion by Johnny9164** [hyperledger-labs/solang#711] [Q&A]
Hi, sorry to bother yall again... Buuutttt I cannot get this command to work no matter how i split it up or google possible solutions or learn the commands. Thanks again yall. 
![Capture](https://user-images.githubusercontent.com/96505092/160043372-13908207-3e55-4b74-b108-09ebec373163.PNG)

**Comment by seanyoung** [comment]
When you run this command, you should have a `build/` directory, and a `contracts/ERC20.sol` file as well. Can you please give us steps to reproduce this problem?

What would be useful is the commands you run and their output.

Thanks!
